packages/lambdas/src/shared/oss-api.ts

```diff
@@ -26,5 +26,39 @@ export function apiClient(apiURL: string) {
         maxAttempts: MAX_API_NOTIFICATION_ATTEMPTS,
       });
     },
+    postConsolidated: async function ({
+      patientId,
+      bundle,
+      requestId,
+      conversionType,
+      resources,
+      dateFrom,
+      dateTo,
+    }: {
+      patientId: string;
+      bundle: medical.SearchSetBundle;
+      requestId?: string;
+      conversionType?: medical.ConsolidationConversionType;
+      resources?: medical.ResourceTypeForConsolidation[];
+      dateFrom?: string;
+      dateTo?: string;
+    }) {
+      const postConsolidated = `${apiURL}/internal/patient/${patientId}/consolidated`;
+      await executeWithNetworkRetries(
+        () =>
+          ossApi.post(postConsolidated, {
+            bundle,
+            requestId,
+            conversionType,
+            resources,
+            dateFrom,
+            dateTo,
+          }),
+        {
+          retryOnTimeout: true,
```

Comment:
I thought the endpoint we're calling here is not idempotent. It could trigger multiple WH notifications, no?

Reply:
Will remove retries here